src/main/java/hudson/tasks/junit/JUnitParser.java

```diff
@@ -48,9 +50,13 @@
 @Extension // see TestResultParser.all
 public class JUnitParser extends TestResultParser {
 
+    private static final Logger LOGGER = Logger.getLogger(JUnitParser.class.getName());
+
     private final boolean keepLongStdio;
     private final boolean allowEmptyResults;
 
+    private final boolean parseOldReports;
```

Comment:
Although I normally advocate for options to be named in the positive rather than negative sense, in this case the behavior is sufficiently surprising that I would suggest it be called `skipOldReports` for example.

Reply:
the name was according to the behaviour I changed back as with `false` as default.
if we choose `skipOldReports` (with `false` as default) so we do not return to the previous behaviour :) 
if there is some compromise for that no problem I will change that.
@jtnord ?